index.css

```
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+}
+
+header {
+  background-color:  #0088F080;
+  color: white;
+  padding: 20px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo img {
```

No veo una clase `.logo`, asi que no tiene sentido eso. Mejor colócale la clase a la imagen
```suggestion
.logo {
```